### PR TITLE
Solved - Kevin Del Castillo Ramirez

### PR DESCRIPTION
**Pull request by quebin31**
### All tests passing ✅

### CODE REVIEW

File: src/calls-cost.js [hunk 1]
```diff
@@ -38,7 +38,91 @@
  * 
  * @returns {CallsResponse}  - Processed information
 */
+function callsCost(calls) {
+    const callsDetails = [];
+    let totalCost = 0;
 
-function callsCost(calls) { }
+    for (const call of calls) {
```
Comment (fhereduardo90) [inline src/calls-cost.js:45]: we use 2 spaces for indentation https://github.com/airbnb/javascript#whitespace

File: src/calls-cost.js [hunk 2]
```diff
@@ -38,7 +38,91 @@
...
+        const processedCall = processCall(call)
+        if (processedCall === null) continue
```
Comment (fhereduardo90) [inline src/calls-cost.js:47]: I'm not against comparing against `null` but remember that `''`, `null`, `false`, and `undefined` are falsy values so you could have done `if (!processedCall)`

File: src/calls-cost.js [hunk 3]
```diff
@@ -38,7 +38,91 @@
...
+            call.duration,
+        )
+    } else if (call.type === 'Local') {
+        callCost = compoundCost(
+            LOCAL_COST,
+            LOCAL_COST, // cost is the same as initial
+            call.duration, // doesn't have a threshold
+            call.duration,
+        )
+    } else {
```
Comment (fhereduardo90) [inline src/calls-cost.js:99]: you could have simplified this by using `switch`

File: src/calls-cost.js [hunk 4]
```diff
@@ -38,7 +38,91 @@
...
+        return null
+    }
+
+    return { callCost: roundDecimal(callCost, 2), ...call }
```
Comment (fhereduardo90) [inline src/calls-cost.js]: spread operator goes first

File: src/morse-code.js [hunk 5]
```diff
@@ -1,13 +1,108 @@
...
+    'T': '-',
+    'U': '..-',
+    'V': '...-',
+    'W': '.--',
+    'X': '-..-',
+    'Y': '-.--',
+    'Z': '--..',
+}
+
+const MORSE_ENGLISH = (() => {
```
Comment (fhereduardo90) [inline src/morse-code.js]:
```
const MORSE_ENGLISH = {
  '': '  ',
  ...Object.entries(ENGLISH_MORSE).reduce((prev, [k, v]) => ({ ...prev, [v]: k }), {})
};

File: src/people.js [hunk 6]
```diff
@@ -26,6 +26,54 @@
  * @returns {PeopleResponse}  - Processed information
 */
 
-function peopleInformation(people) { }
+function peopleInformation(people) {
+    if (people.length === 0) {
```
Comment (fhereduardo90) [inline src/people.js:30]: `if (!people.length)`

File: src/people.js [hunk 7]
```diff
@@ -26,6 +26,54 @@
...
+    return people.reduce(accProm, [0, 0])
+}
+
+/**
+ * Finds the youngest and tallest person in {@link people}.
+ * @param {Array<Person>} people People information to be processed
+ * @returns {[Person, Person]} the youngest and tallets person.
+ */
+function findYoungestAndTallest(people) {
+    let youngest = people[0], tallest = people[0]
```
Comment (fhereduardo90) [inline src/people.js]: https://github.com/airbnb/javascript#variables--one-const

Comment (fhereduardo90) [reply src/people.js]: https://github.com/airbnb/javascript#variables--const-let-group

File: src/template.js [hunk 8]
```diff
@@ -19,6 +19,29 @@
...
 */
 
-function template(model, character, message) { }
+function template(model, character, message) {
+    if (!model.includes(character) || message.length === 0) {
+        return "Invalid parameters"
+    }
+
+    let index = 0
+    let formatted = ""
```
Comment (fhereduardo90) [inline src/template.js]: https://github.com/airbnb/javascript#strings--quotes

File: src/fibonacci.js [hunk 9]
```diff
@@ -6,6 +6,21 @@
...
 
-function fibonacci(limit) { }
+function fibonacci(limit) {
+    const numbers = []
+
+    let a = 0
+    let b = 1
+    let t = 1
+
+    while (numbers.length != limit) {
```
Comment (fhereduardo90) [inline src/fibonacci.js]: https://github.com/airbnb/javascript#comparison-operators--equality

File: src/calls-cost.js (hunk 3 above)
Comment (quebin31) [reply src/calls-cost.js:99]: I was thinking about that, I'm actually a big fan of pattern matching but not a big fan of the `switch` with C style, but nonetheless both work the same

File: src/calls-cost.js (hunk 2 above)
Comment (quebin31) [reply src/calls-cost.js:47]: Yes, that's something I remember from another languages like C or C++, my take on using `null` is verbosity and readability, but yeah both are fine.

File: src/calls-cost.js (hunk 1 above)
Comment (quebin31) [reply src/calls-cost.js:45]: Oh, didn't realize that, but that's something that can be easily fixed in any project using `.editorconfig`

File: src/calls-cost.js (hunk 4 above)
Comment (quebin31) [reply src/calls-cost.js]: Fixed

File: src/fibonacci.js (hunk 9 above)
Comment (quebin31) [reply src/fibonacci.js]: Oops! This was not intentional, fixing it.

File: src/template.js (hunk 8 above)
Comment (quebin31) [reply src/template.js]: Fixing it.

File: src/people.js (hunk 7 above)
Comment (quebin31) [reply src/people.js]: Fixing it.

File: src/people.js (hunk 6 above)
Comment (quebin31) [reply src/people.js:30]: Yes, this more about a verbosity thing I like to do that I learned from another languages which are more strict about this, I would even prefer a method that would tell me this is empty.

**Pull request closed without merging**